src/main/java/edu/ie3/datamodel/utils/validation/SystemParticipantValidationUtils.java

```diff
@@ -0,0 +1,429 @@
+   * properties
+   *
+   * @param chpInput ChpInput to validate
+   */
+  public static void checkChp(ChpInput chpInput) {
+    checkChpType(chpInput.getType());
+  }
+
+  /**
+   * Validates a chpTypeInput if: <br>
+   * - common system participants values (capex, opex, sRated, cosphiRated) are valid <br>
+   * - its efficiency of the electrical inverter is between 0% and 100% <br>
+   * - its thermal efficiency of the system is between 0% and 100% <br>
+   * - its rated thermal power is positive <br>
+   * - its needed self-consumption is not negative
+   *
+   * @param chpTypeInput ChpTypeInput to validate
+   */
+  public static void checkChpType(ChpTypeInput chpTypeInput) {
+    checkType(chpTypeInput);
+    detectNegativeQuantities(new Quantity<?>[] {chpTypeInput.getpOwn()}, chpTypeInput);
+    detectZeroOrNegativeQuantities(new Quantity<?>[] {chpTypeInput.getpThermal()}, chpTypeInput);
+    if (chpTypeInput.getEtaEl().getValue().doubleValue() < 0d
+        || chpTypeInput.getEtaEl().getValue().doubleValue() > 100d)
+      throw new InvalidEntityException(
+          "Efficiency of electrical inverter of CHP unit type must be between 0% and 100%",
+          chpTypeInput);
+    if (chpTypeInput.getEtaThermal().getValue().doubleValue() < 0d
+        || chpTypeInput.getEtaThermal().getValue().doubleValue() > 100d)
+      throw new InvalidEntityException(
+          "Thermal efficiency of system of CHP unit type must be between 0% and 100%",
+          chpTypeInput);
+  }
+
+  /**
+   * Validates a EvInput if: <br>
+   * - {@link SystemParticipantValidationUtils#checkEvType(EvTypeInput)} confirms a valid type
+   * properties
+   *
+   * @param evInput EvInput to validate
+   */
+  public static void checkEv(EvInput evInput) {
+    checkEvType(evInput.getType());
+  }
+
+  /**
+   * Validates a EvTypeInput if: <br>
+   * - common system participants values (capex, opex, sRated, cosphiRated) are valid <br>
+   * - its available battery capacity is positive <br>
+   * - its energy consumption per driven kilometre is positive
+   *
+   * @param evTypeInput EvTypeInput to validate
+   */
+  public static void checkEvType(EvTypeInput evTypeInput) {
+    checkType(evTypeInput);
+    detectZeroOrNegativeQuantities(
+        new Quantity<?>[] {evTypeInput.geteStorage(), evTypeInput.geteCons()}, evTypeInput);
+  }
+
+  /**
+   * Validates a FixedFeedInInput if: <br>
+   * - its rated apparent power is not negative <br>
+   * - its rated power factor is between 0 and 1
+   *
+   * @param fixedFeedInInput FixedFeedInInput to validate
+   */
+  public static void checkFixedFeedIn(FixedFeedInInput fixedFeedInInput) {
+    detectNegativeQuantities(new Quantity<?>[] {fixedFeedInInput.getsRated()}, fixedFeedInInput);
+    if (fixedFeedInInput.getCosPhiRated() < 0d || fixedFeedInInput.getCosPhiRated() > 1d)
+      throw new InvalidEntityException(
+          "Rated power factor of fixed feed-in unit must be between 0 and 1", fixedFeedInInput);
```

Comment:
done -> method takes a `InputEntity` and a `double` as input so it can be applied to all class types here (`AssetInput` as well as `AssetTypeInput`) -> is that a good solution?